assets/js/monitor.js

```diff
@@ -32,8 +37,9 @@ jQuery(async function () {
     }
 
     const dropDownText = $(`#${dropDownType} option:selected`).text()
-    const dropDownTypeIdValue = Number.parseInt($(`#${dropDownType}`).val())
-    const dropDownTypeId = Number.isNaN(dropDownTypeIdValue) ? 0 : dropDownTypeIdValue
+    const dropDownTypeIdValue = $(`#${dropDownType}`).val()
+    const parsedId = parseInt(dropDownTypeIdValue, 10)
+    const dropDownTypeId = !dropDownTypeIdValue || isNaN(parsedId) ? 0 : parsedId
```

Comment:
What I'm saying is, this does the same job in more lines of code? I don't understand why you've changed it?